draft-ietf-acme-acme.md

```diff
@@ -2688,6 +2691,23 @@ within their trusted network and use these resolvers both for both CAA record
 lookups and all record lookups in furtherance of a challenge scheme (A, AAAA,
 TXT, etc.).
 
+## Recovery From Host Compromise
+
+If a host is compromised, once the owner regains exclusive control of the host,
+they will want to take steps to revoke any certificates the attacker may have
+issued, and to ensure that the attacker cannot issue new certificates for the
+affected hostnames.
+
+If the attacker gained access to the private key for an ACME account, but has
+not yet changed the ACME account keypair, the proper owner can use the key
+rotation functionality to regain exclusive control of their ACME account.
+
+If the attacker did change the ACME account keypair, or if the attacker created
+their own ACME account and authorized the hostname, the proper owner, after
+regaining control of the host, can authorize that hostname on their own account
```

Comment:
I think some potential ambiguity could be removed by saying "can authorize that hostname on their own *new* account" here.